### ISSUE TITLE
Wrong terminology: logits are not log probabilities in code.

### ISSUE THREAD

**Issue by rumschuettel** [mozilla/DeepSpeech#2091]
https://github.com/mozilla/DeepSpeech/blob/c1816d6dfc80b8f9ab13cbb94e4e79dce3cf17f5/DeepSpeech.py#L608

I don't want to be pedantic, but why are these variables called "logits", if they are not log probabilities? Even if the previous layers were to produce logits (which they don't, I never see a log call) or the rnn outputs logits (unclear), what's the purpose of taking a `softmax` of logits? Just to normalize them to a range [0,1]?

Thanks for any pointers.
~ Johannes

**Comment by kdavis-mozilla** [comment]
Logits is the accepted term used for the input to Tensorflow's  [softmax](https://www.tensorflow.org/api_docs/python/tf/nn/softmax). Quoting the documentation:
```python
tf.nn.softmax(
    logits,
    axis=None,
    name=None,
    dim=None
)
```
As for the outputs of the softmax, yes logit is an inappropriate choice of name. You can suggest a more appropriate name in a PR.

**Comment by rumschuettel** [comment]
Cool. I was simply confused as when I inspected the trained model in tensorboard, the "logits:0" layer does not, in fact, contain logits, but probabilities.

Thanks!!
